ultravox/training/configs/meta_config.yaml

```diff
@@ -1,7 +1,7 @@
 text_model: "meta-llama/Meta-Llama-3-8B-Instruct"
 audio_model: "facebook/wav2vec2-base-960h"
 
-data_sets: ["gigaspeech"]
```

Comment:
epoch training does not work with non generic datasets. Removing this so that we can specify all datasets from release_config

Reply:
I see `max_steps` is defined and non-zero, so we're not in epoch mode.

Setting datasets to be empty means the code won't work unless further configs are specified.
Even though most training uses `release_config`, its presence is not a must, so we cannot rely on it.